src/main/java/com/scalar/database/transaction/consensuscommit/Snapshot.java

```diff
@@ -19,7 +19,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-/** */
+/** A class to manage transaction isolation (snapshot or serial) */
```

Comment:
It's not correct.  Please search for `snapshot isolation` to understand what it does. Also, Akon's paper is helpful.

Reply:
Ah, sorry about that. I read about snapshot isolation in Kleppmann's book, but then I saw that a constructor allows you to specify an isolation level. However, I couldn't completely understand what `toSeriablizable()` was doing.